internal/constants/scripts.go

```diff
@@ -43,163 +45,183 @@ fi
 # A map of host mac addresses to interface names
 declare -A host_macs_to_hw_iface
 
-# The directory that contains nmconnection files for the current host
-host_dir=""
-
-# The mapping file of the current host
-mapping_file=""
+# A mac-to-nic map created from the mapping file associated with the host
+declare -A logical_mac_nic_map
 
-# A nic-to-mac map created from the mapping file associated with the host
-declare -A logical_nic_mac_map
+# Working directory used for storage of processed files before installation in the target directory
+work_dir=$(mktemp -d)
```

Comment:
When running tests does this get cleaned up?

Reply:
Same for the other `mktemp` calls in this file.

Reply:
Here it does https://github.com/openshift/assisted-service/blob/10d0a94f3d6164167056f1f95d2c5cce97705336/internal/constants/scripts.go#L224

Reply:
Others are cleaned too